packages/transaction-controller/src/utils.ts

```diff
@@ -31,6 +31,7 @@ const NORMALIZERS: { [param in keyof TransactionParams]: any } = {
   estimatedBaseFee: (maxPriorityFeePerGas: string) =>
     addHexPrefix(maxPriorityFeePerGas),
   type: (type: string) => (type === '0x0' ? '0x0' : undefined),
+  estimateGasError: (estimateGasError: string) => estimateGasError,
```

Comment:
This is added to make tests pass for the `addTransaction => populates estimateGasError if gas calculation fails on`